dali/test/python/test_dali_cpu_only.py

```diff
@@ -60,6 +60,20 @@ def get_data():
     return out
 
 
+# The same code is used as CPU-only pipeline to test if TF plugin loads successfully
+# during its installation.
+def test_tensorflow_build_check():
```

Comment:
Is this related to the PR?

Reply:
It is a bit related, because I had to run this code to see where it is failing :P